migrations/ribbon_eth_covered_call_v2.js

```diff
@@ -0,0 +1,87 @@
+const RibbonThetaVault = artifacts.require("RibbonThetaVault");
```

Comment:
We're no longer using Truffle anymore so we can't do `artifacts.require`, and do migrations.
It's more vanilla and we have to manually deploy with typescript files now